typescript/sdk/src/test/envSubsetDeployer/README.md

```diff
@@ -1,8 +0,0 @@
-# Environment Subset Deployer App
```

Comment:
Are you sure you want to delete this test app? I'm not seeing much deployment test coverage without it